ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -197,8 +197,8 @@ public void sendRequest(
       requestBuilder.method(method, multipartBuilder.build());
     } else {
       // Nothing in data payload, at least nothing we could understand anyway.
-      // Ignore and treat it as if it were null.
-      requestBuilder.method(method, null);
+      // Creating empty requestBody to solve issue similar to #3371
```

Comment:
@lexs I had updated the comment with "similar to #3371", so that others get the context why we are doing this change.